store.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+</head>
+<body>
+<header>
+    <img src="img/karma-logo.svg">
+    <div class="nav-header" id="nav-header">
+        <a href="index.html">Meet Karma</a>
+        <a href="#">How it Works</a>
+        <a href="store.html">Store</a>
+        <a href="#">Blog</a>
+        <a href="#">Help</a>
+        <a href="#">Login</a>
+    </div>
+    <div class="burger-button" id="burger-button">
+        <span class="burger-icon">&#9776;</span> <!-- Burger icon -->
+    </div>
+</header>
+
+<section class="container">
+<div class="element1">
+<h1>Order your Karma Wifi Device Today!</h1>
+<form>
+<div class="form">
+<div class="input-group">
+    <label for="first-name">FirstName*</label>
+    <input type="text" id="first-name" name="first-name" placeholder="Enter your first name">
+</div>
+<div class="input-group">
+    <label for="last-name">LastName*</label>
+    <input type="text" id="last-name" name="last-name" placeholder="Enter your last name">
+</div>
+</div>
+<div class="form-address">
+<div class="address11">
+    <label for="address">Address*</label>
+    <input type="text" id="address1" name="address1" placeholder="Enter your first address">
+</div>
+<div class="address12">
+    <label for="address2">Address2</label>
+    <input type="text" id="address2" name="address2" placeholder="Enter your second address">
+</div>
+</div>
+<div class="form">
+    <div class="input-group">
+        <label for="city">City</label>
+        <select id="city" name="city">
+            <option value="Spain">Spain</option>
+            <option value="los-angeles">Los Angeles</option>
+            <option value="France">France</option>
+            <option value="France"> Italy</option>
+            <option value="France">Japan</option>            
+        </select>
+    </div>
+    <div class="input-group">
+        <label for="postal-code">Postal Code</label>
+        <input type="text" id="postal-code" name="postal-code" placeholder="Enter postal code">
+    </div>
+</div>
+
+ <legend>Select a Color:</legend>
+<div class="radio-group">
+    <div class="radio">
+        <input type="radio" id="orange" name="orange1" value="orange">
```

Comment:
Only one color can be selected, right?
You need to use the same name for all the color inputs. "color" for example

Reply:
"Apologies, it seems the dropdown list for colors was deleted from the code. No worries, I’ll add it back again."